packages/core/test/invoke.test.ts

```diff
@@ -715,15 +715,15 @@ describe('invoke', () => {
         }
       });
 
-      const expectedStateValue = 'two';
       let currentState: AnyState;
```

Comment:
This would be even easier to follow:
```ts
      const actor = interpret(mainMachine).start();

      expect(actor.getSnapshot().value).toBe('two');
```

Note that this also requires removing the `done` callback argument